test/turbopack-build-tests-manifest.json

```diff
@@ -2881,12 +2881,13 @@
     },
     "test/e2e/app-dir/parallel-routes-catchall/parallel-routes-catchall.test.ts": {
       "passed": [
-        "parallel-routes-catchall should match both the catch-all page & slot",
         "parallel-routes-catchall should match correctly when defining an explicit page & slot",
-        "parallel-routes-catchall should match correctly when defining an explicit page but no slot",
         "parallel-routes-catchall should match correctly when defining an explicit slot but no page"
       ],
-      "failed": [],
+      "failed": [
```

Comment:
x-ref: https://vercel.slack.com/archives/C03EWR7LGEN/p1726286012346389

Reply:
This is not a bug of tree shaking. This should not pass, but previously it passed as a result of a bug.